### PR TITLE
Vortex: improve HU mode-switch, keep-alive, handle brake errors

### PR DESCRIPTION
**Pull request by switchabl**
Several small improvements for Tacx Vortex:
- improve reliability of switching HU to PC-mode: keep sending mode switch command whenever the
  wrong mode is detected (e.g. first command gets lost, HU was turned off and on again)
- send keep-alive page every 10s (like TTS4) instead of every 0.25s; should be more than enough
  (head unit only turns off after 3 minutes)
- handle and display brake errors/warnings
- improve debug output for Vortex pages (now consistent with Genius and Bushido output)
- disable time-out for pairing to the brake
- replace "i-Vortex" with "Vortex" everywhere for consistency:
  i-Vortex is just the T2170 bundle, but older T1960/T1965 bundles should work just
  as well
- minor code clean-up

Windows binary: https://github.com/switchabl/FortiusANT/suites/2021188431/artifacts/40537444

Tested with my T2171 brake/T1962 HU, will try to get other Vortex users to test before merge.

Not sure if this helps with #219, still looking into that (could not reproduce with my Vortex).

### CODE REVIEW

**Comment by WouterJD** [comment]
> will try to get other Vortex users to test before merge

OK; I will wait for GO before merge.
Code looks good. Thanks.

**Comment by cermatej** [comment]
related issues: #219 #215

Using updated FW (HU 0.2.5, brake 0.2.7) and provided .exe build seemed to fixed most of the problems:

-  **HU not being able to enter PC mode**: tested several times with different scenarios and worked flawlessly
-  **not mentioned bug with Garmin watches**: (not critical at all but nice to have) I was using Garmin watches to track my workouts with ANT+ output from FortiusANT and the maximum watt values were sometimes too high for a few seconds (around 10k Watts) thus ruining the output data. Now the data seems to be precise as the FortiusANT reported.
-  **Powerdrops**: used to experience those only in manual mode (FE-C worked ok even before), when tested did not experience any when running Zwift both in ERG mode and normal

Note:
- **Power increase**: experienced quite notable increase of power output. I try to calibrate brake with HU before every ride. As long as it increased I am happy :) (I tried to push it and my FTP increased by about 50W so I dont think its the matter of better effort) but I have no other option to compare the values with. Will calibrate before every ride and report the results.

In conclusion it seems that all the major bugs were fixed as reported in #219 by @ditisdirkdewit as well. Thanks!

**Comment by switchabl** [comment]
@cermatej Thanks for your feedback, that is good to hear!

>     * **Power increase**: experienced quite notable increase of power output. I try to calibrate brake with HU before every ride. As long as it increased I am happy :) (I tried to push it and my FTP increased by about 50W so I dont think its the matter of better effort nor bad calibration) but I have no other option to compare the values with. Will calibrate before every ride and report the results.

The calibrated power value (in watts) comes directly from the brake, we don't do any post-processing on that (and we don't send any calibration commands either), so I don't think this is because of the patch. Maybe they have changed/improved the calibration algorithm in the newer firmware. The power from my Vortex (with the latest firmware, after calibrating with the head unit) is very close to my Garmin Vector 2 pedals.

**Comment by switchabl** [comment]
@WouterJD Both @cermatej and @ditisdirkdewit have tested this successfully and I have used it a bit myself without any problems, so I think this can be merged.

**Comment by WouterJD** [comment]
I see this can be merged into Master.
What is the correct procedure to rebase to Raspberry_Release6, which is quite an update. Is the manual checking the right way to do?

**Comment by switchabl** [comment]
@WouterJD I see. I will rebase to Raspberry_Release6. I don't expect any actual conflicts, probably just some comments/changelog so this should be easy hopefully.

**Comment by WouterJD** [comment]
Thanks for the rebasing; I will merge as soon as available
